src/main/steps/c100-rebuild/child-details/util.ts

```diff
@@ -30,12 +30,43 @@ export const getDataShape = (): ChildrenDetails => ({
   },
 });
 
+export const getOtherChildDataShape = (): OtherChildrenDetails => ({
+  id: uuidv4(),
+  firstName: '',
+  lastName: '',
+  personalDetails: {
+    dateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    isDateOfBirthUnknown: YesNoEmpty.EMPTY,
+    approxDateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    gender: Gender.EMPTY,
+    otherGenderDetails: '',
+  },
+});
+
 export const getChildDetails = (children: ChildrenDetails[] | [], childId: string): ChildrenDetails | undefined =>
   children.find(child => child.id === childId);
 
+export const getOtherChildDetails = (
```

Comment:
Why we need another method when getChildDetails can be tweaked to be reused across child-details & other children? Please tweak the getChildDetails so that it can be reused.

Reply:
okay just seen this, will fix when I get back